static/js/loading.ts

```diff
@@ -41,14 +41,8 @@ export function make_indicator(
     $container.append($spinner_elem);
     let text_width = 0;
 
-    if (text !== undefined) {
-        const $text_elem = $("<span>").addClass("loading_indicator_text");
-        $text_elem.text(text);
-        $container.append($text_elem);
-        // See note, below
-        if (!abs_positioned) {
-            text_width = 20 + ($text_elem.width() ?? 0);
-        }
+    if (text !== undefined && !abs_positioned) {
+        text_width = 20;
```

Comment:
This function is called in dozens of places; removing this will break any others using the `text` option. 

What you want to do is likely change the parameters passed from the `setup.js` call point to this function.